docs/guides/tools.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Tools
+---
+
+# Tools
+
+Strawberry provides some utility functions to help you build your GraphQL
+server. All tools can be imported from `strawberry.tools`
+
+---
+
+### `create_type`
+
+Create a Strawberry type from a list of StrawberryFields.
```

Comment:
```suggestion
Create a Strawberry type from a list of fields.
```

what do you think of renaming `StrawberryFields` to fields here? I think someone that's new to the library might not know what a `StrawberryField` is

Reply:
Staying agnostic to the implementation here seems reasonable to me

Reply:
Yep makes sense.